ui/src/app/collection/detail/manage-collection.component.ts

```diff
@@ -140,11 +142,13 @@ export class ManageCollectionComponent extends SkillsListComponent implements On
         icon: this.addIcon,
         primary: !this.collectionHasSkills, // Primary only if there are no skills
         callback: () => this.addSkillsAction(),
+        visible: () => this.authService.isEnabledByRoles("COLLECTION_SKILLS_UPDATE")
```

Comment:
I thought we had these actions as variables/constants/symbols some place, rather than just as string literals. If we have them declared as constants, can we please use those constants everywhere?

Reply:
Should be the following, I think.
`isEnabledByRoles(ACTION_ROLES.COLLECTION_SKILLS_UPDATE)`

Reply:
And 4 more times below.